calico_containers/tests/unit/netns_test.py

```diff
@@ -85,3 +85,35 @@ def test_veth_exists_false(self, m_check_call, m_open):
                                              stderr=ANY,
                                              stdout=ANY)
 
+
+class TestRoute(unittest.TestCase):
+    def test_metric(self):
+        """
+        Test that a Route object correctly parses the metric of a route
+        """
+        self.assertEqual(Route("default via 172.24.114.1 dev eth0").metric, 0)
+        self.assertEqual(Route("default via 172.24.114.1 dev eth0 metric 0").metric, 0)
+        self.assertEqual(Route("172.17.0.0/16 dev eth0 metric 1").metric, 1)
+        self.assertEqual(Route("172.17.0.0/16 dev eth0 metric 240").metric, 240)
+
+    def test_default(self):
+        """
+        Test that a route object correctly flags if the route is a default route or not
+        """
+        self.assertTrue(Route("default via 172.24.114.1 dev eth0 metric 1").default)
+        self.assertFalse(Route("172.17.0.0/16 dev eth0 metric 1").default)
+
+    def test_increment_metric(self):
+        """
+        Test that a route object correctly returns an incremented metric route.
+        """
+        route = Route("default via 172.24.114.1 dev eth0")
+        self.assertEqual(str(route), "default via 172.24.114.1 dev eth0 metric 0")
+
+        route.metric += 1
+        self.assertEqual(str(route),
+                         "default via 172.24.114.1 dev eth0 metric 1")
+
+        route.metric += 1
+        self.assertEqual(str(route),
+                        "default via 172.24.114.1 dev eth0 metric 2")
```

Comment:
I think you need to add one more test to test your MAX metric code.  Otherwise LGTM